Adeolu_form.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Intro to HTML and Javascript</title>
+  <link href="style.css" rel="stylesheet" type="text/css" />
+</head>
+
+<body>
+ 
+    <form>
+<b><label>First_Name:</label>  </b>
```

Comment:
No need for an underscore when setting the label text. A visitor to your website will see the label text, so it should be normal text: "First Name"